script/utils/Constants.sol

```diff
@@ -18,6 +20,12 @@ library ArcadiaLendingSafes {
 }
 
 library InterestRateParameters {
+    // ToDo
+    uint72 internal constant BASE_RATE_CBBTC = 4 * 1e16; // 4%
```

Comment:
seems a bit high, no?

Reply:
Is deployed like that for now, but we have a ticket in backlog to review all pool setting next week.